### ISSUE TITLE
Scroll with wheel while multiple items selected

### ISSUE THREAD

**Issue by PawnM0v3** [BLCM/OpenBLCMM#60]
The ability to use your scroll wheel while having multiple items selected, be it a few set commands or entire folders. Makes some work slow having to drag the items to the top of the screen and wait for it to scroll up slowly.

**Comment by apocalyptech** [comment]
Yeah, I agree this would be a good idea; thanks for the submission!  As it stands, that probably relies on some Java input shenanigans which I'm not feeling up to trying to investigate at the moment, but hopefully I'll get to it for a future release.  :)
